app/models/announcement.rb

```diff
@@ -0,0 +1,5 @@
+class Announcement < ActiveRecord::Base
+  def token
+    "#{self.class.name.downcase}_#{created_at.iso8601}"
```

Comment:
What is the reason we are parsing the `created_at` to a specific iso standard? Did the default rails `created_at` give issues?